plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import os, sys
+import urllib, urlparse, json
+import xbmcgui, xbmcplugin, xbmcaddon
+import requests
+from register import device
```

Comment:
I was away for most of the week, so I didn't manage to answer earlier. simplecache is now in place of memstorage.py, so I'll lose both - memstorage.py and device.py where last will be transformed in the class within the main script (addon will have single py script) and init check will be called depending on tmp_store object timeout every x hours since simplecache doesn't support store in memory, otherwise it would be called every time kodi + addon starts. Am I on the right track here? 

Btw, with regards to your comments below, everything should be OK now but I didn't push it yet in order to minimise the number of commits.